extensions/amp-story/amp-story-animation.md

```diff
@@ -0,0 +1,648 @@
+---
+$category@: presentation
+formats:
+  - stories
+teaser:
+  text: An animation effect definition for an AMP story.
+---
+
+<!--
+Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# amp-story-animation
```

Comment:
Mostly a dupe of [`amp-animation.md`](https://go.amp.dev/c/amp-animation) per @CrystalOnScript's suggestion.